spec/board_spec.rb

```diff
@@ -50,5 +50,18 @@
       board.add_marker('O', 9)
       expect(board.winner).to eql(false)
     end
+
```

Comment:
I think 'X' start first so he has to have at least 5 moves, what you can do to fix that is switch 'X' and 'O' and change the order in the code so to appear like 'X' choose a position and then 'O' choose another one